images/spec/support/spec_helper.rb

```diff
@@ -17,3 +17,7 @@ def ensure_on(path)
 RSpec.configure do |c|
   c.alias_it_should_behave_like_to :it_has_behaviour, 'has behaviour:'
 end
+
+def uri_filename(url)
```

Comment:
it feels like this method shouldn't be defined here as it's only used once (in `images/spec/models/refinery/image_spec.rb` and could be used inline:

```ruby
it "contains its filename" do
  expect(File.basename(URI.parse(created_image.url)))
    .to match(/\A#{created_image.image_name}/)
end
```

Reply:
And done.  I must have used it in a few places (but perhaps in different gems, because I'm sure I just picked it out of another spec_helper.